tests/workflow/interfaces/qnode/test_jax_jit_qnode.py

```diff
@@ -189,10 +166,7 @@ def circuit(a, b, c):
             qml.RX(c + c**2 + jax.numpy.sin(a), wires=0)
             return qml.expval(qml.PauliZ(0))
 
-        res = jax.grad(circuit, argnums=[0, 2])(a, b, c)
-
-        if diff_method == "finite-diff":
-            assert circuit.qtape.trainable_params == [0, 2]
+        res = jax.jit(jax.grad(circuit, argnums=[0, 2]))(a, b, c)
```

Comment:
So here (and below) we were not really testing `jax.jit`, although the name of the file is `test_jax_jit_qnode`. lol